src/form/Select/Select.tsx

```diff
@@ -70,6 +70,11 @@ export interface SelectProps {
    */
   value?: string | string[];
 
+  /**
+   * The deafult value of the input filter.
+   */
+  defaultFilterValue?: string;
```

Comment:
On the fence whether it should be initialFilterValue or leave as defaultFilterValue. Do we even need to add value suffix ?